vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
There are two basic responses to these comments, and I'll try and address each of your concerns.

The first issue is "should we use 1622.2 enumerations?" I believe we should for the sake of compatibility and the increased likelihood that we'll get more data as a result of this compatibility.

The second issue is "does 1622.2 contain the right set of enumerations?" And here I agree that this list could use work.

> For example, what is the difference between city, local, locality, and municipality (and county in the case of a City and County like San Francisco)?

WRT city versus local or locality or municipality, I can see the former used when there's an actual city, and the latter used when it's a sub-city district or a township. I agree, however, that these enumerations may be overly specific and are probably due to someone in 1622.2 saying that "well, we don't have *local* elections, we have *locality* elections, so that needs to be separate" rather than recognizing that these are "close-enough" labels.

> FWIW, I did find what seem to be at least three types of judicial districts in California, each with its own numbering sequence (presumably all applicable in retention elections).

I think this echoes our earlier conversation about offices versus the governmental body to which those offices are related. The enumeration should *only* establish the *type* of the office, not attempt to define the overall structure of government. This distinction should go in the eventual "Best Practices" document, as this would be a bad object:
```
  <ElectoralDistrict>
    <Name>Fourth District</Name>
    <Type>judicial</Type>
  </ElectoralDistrict>
```
whereas this would be better
```
  <ElectoralDistrict>
    <Name>California Court of Appeals, Fourth District</Name>
    <Type>judicial</Type>
  </ElectoralDistrict>
```

> my original suggestion wasn't to invent a new identification scheme, but to introduce a parent object that would serve to group the districts within a numbering sequence.

My concern here is that this simply moves the disambiguation problem up a level and doesn't necessarily help us with the underlying "feed publisher includes insufficient data" problem. Someone could still create an ElectoralDistrict named "Second District" of type "judicial" and have its parent be "Fourth District" (also of type "Judicial") and we're no closer to actually figuring out how to disambiguate and resolve these problems.

I propose we

1) Include the current set of 1622.2 enumerations as-is; and
2) Create a concrete list of suggestions for the 1622.2 schema (e.g., "either document specific examples and provide guidelines on how to select city vs locality vs municipality, or drop possibly-duplicate values") and then track and resulting changes.